node/core/approval-voting/src/lib.rs

```diff
@@ -1104,6 +1259,11 @@ async fn handle_from_overseer(
 				}
 			}
 
+			if !state.recovery_state.complete() {
```

Comment:
I don't see where `state.recovery_state` is modified, could you point me to that?

Reply:
Just saw this. In the implementation of `fn complete(&mut self)`, we assign to the `state.recovery_state` variable, i.e. 

`
fn complete(&mut self) -> bool {
        let complete = *self == RecoveryState::Complete;
        *self = RecoveryState::Complete;
        complete
}
`

Reply:
Oh, I see. Missed that `fn complete` modifies the state.